### DISCUSSION TITLE
Using Helpers to input values in Mini Graph Card settings

### DISCUSSION THREAD

**Discussion by yesterdayshero** [kalkih/mini-graph-card#791] [Q&A] [answered]
I was trying to use some Helpers to make it easier to change values across a group of Mini Graph Cards. For example, instead of needing to go into each card and changing the hours_to_show, I could use a Helper to input the value and then just change the state of the Helper if I want to see a different number of hours across multiple graphs.

`hours_to_show: {{ states('input_number.graphs_hours_to_show') | int }}`

It doesn't seem to work with the above implementation, but is there a way to make these values dynamic?

**Comment by ildar170975** [comment, marked as answer]
1. Currently you can use `config-template-card`.
2. Start monitoring this issue: https://github.com/kalkih/mini-graph-card/issues/697
